eclipse_plugin/build.properties

```diff
@@ -0,0 +1,7 @@
+source.. = src/
+output.. = target/classes
+bin.includes = META-INF/,\
+               .,\
+               plugin.xml,\
+               lib/google-java-format-1.1-SNAPSHOT.jar,\
```

Comment:
@elharo How do I refer to the `google-java-format-XXX.jar` artifact just created prior to this module? Without losing the possibilty to build the plug-in in Eclipse PDE itself.